tests/test_decodepsbtqr.py

```diff
@@ -297,6 +297,9 @@ def test_bitcoin_address():
     
     main_bech32_address2 = "bitcoin:bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq?amount=12000"
     main_bech32_address3 = "BITCOIN:bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq?junk"
+    main_bech32_address4 = "bitcoin:BC1QAR0SRRR7XFKVY5L643LYDNW9RE59GTZZWF5MDQ?junk"
+    main_bech32_address5 = "BITCOIN:BC1QAR0SRRR7XFKVY5L643LYDNW9RE59GTZZWF5MDQ?junk"
+    main_bech32_address6 = "BC1QAR0SRRR7XFKVY5L643LYDNW9RE59GTZZWF5MDQ"
```

Comment:
Suggest refactoring to something like:

```python
bech32_capitalization_variations = [
    "bitcoin:...",
    "BITCOIN:...",
]
```

And simplifying the `for`:
```python
for capitalization_variation in bech32_capitalization_variations:
    ...
```

The `main_bech32_address2` test down on lines 358-362 can be removed since it's now redundant. With that removed, there's no need for the var name.